### PR TITLE
['digest'] issue and ['args'] issue fix

### PR DESCRIPTION
**Pull request by tanaygodse**
Removed digest from model, hardcoded a value 'nextgen' for now, removed 'args' from the payload value

## Proposed Changes

_[describe the changes here...]_

## Linked Issues

https://github.com/fetchai/ai-engine-sdk-python/issues/18
https://github.com/fetchai/ai-engine-sdk-python/issues/19

## Types of changes

_What type of change does this pull request make (put an `x` in the boxes that apply)?_

- [x] Bug fix (non-breaking change that fixes an issue).
- [ ] New feature added (non-breaking change that adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to stop working as expected).
- [ ] Documentation update.
- [ ] Something else (e.g., tests, scripts, example, deployment, infrastructure).

## Checklist

_Put an `x` in the boxes that apply:_

  - [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) guide
  - [x] Checks and tests pass locally

### If applicable

  - [ ] I have added tests that prove my fix is effective or that my feature works
  - [ ] I have checked that code coverage does not decrease
  - [ ] I have added/updated the documentation

### CODE REVIEW

**Comment by tanaygodse** [comment]
@qati Please check

File: ai_engine_sdk/client.py [hunk 1]
```diff
@@ -289,8 +289,8 @@ async def get_messages(self) -> List[ApiBaseMessage]:
                             'id': message['message_id'],
                             'timestamp': message['timestamp'],
                             'text': agent_json['text'],
-                            'model': agent_json['context_json']['digest'],
-                            'payload': agent_json['context_json']['args'],
+                            'model': "nextgen",#agent_json['context_json']['digest'],
```
Comment (qati) [inline ai_engine_sdk/client.py:292]: Can we remove the whole model field from the model?

**Review by qati — APPROVED** [review]

File: ai_engine_sdk/client.py [hunk 2]
```diff
@@ -92,12 +92,14 @@ async def make_api_request(
     }
 
     async with aiohttp.ClientSession() as session:
-        logger.debug(f"\n\n 📤 Request triggered : {method} {api_base_url}{endpoint}")
+        logger.debug(f"\n\n 📤 Request triggered : {
```
Comment (XaviPeiro) [inline ai_engine_sdk/client.py:95]: Please, leave as it was. 
We have max length up to 120 chars so far (standard tends to be 80)

Comment (XaviPeiro) [reply ai_engine_sdk/client.py:95]: probably just an auto-reformat from you tool of your choice. It is in many places, please, check all of them.

**Review by XaviPeiro — CHANGES_REQUESTED** [review]

**Review by XaviPeiro — CHANGES_REQUESTED** [review]
Please, add tests that reflect these fixes.

(For further occasions: let's do that on branches instead of forks, please.)

File: ai_engine_sdk/client.py (hunk 2 above)
Comment (qati) [reply ai_engine_sdk/client.py]: @XaviPeiro we have back set up and configured on this repo? If not lets add that.

Comment (tanaygodse) [reply ai_engine_sdk/client.py]: Fixed the 120 characters formatting.

**Comment by tanaygodse** [comment]
> Please, add tests that reflect these fixes.
> 
> (For further occasions: let's do that on branches instead of forks, please.)

I followed the instructions on https://github.com/fetchai/ai-engine-sdk-python/blob/master/CONTRIBUTING.md

Will create a branch directly for this repository and try again.

**Comment by tanaygodse** [comment]
@XaviPeiro I tried creating a branch in the main repository instead of fork, got this error:
git push upstream digest-issue
ERROR: Permission to fetchai/ai-engine-sdk-python.git denied to tanaygodse.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.

I believe I do not have the rights to make changes to the main repository. Let me know what to do.

**Comment by XaviPeiro** [comment]
> @XaviPeiro I tried creating a branch in the main repository instead of fork, got this error: git push upstream digest-issue ERROR: Permission to fetchai/ai-engine-sdk-python.git denied to tanaygodse. fatal: Could not read from remote repository.
> 
> Please make sure you have the correct access rights and the repository exists.
> 
> I believe I do not have the rights to make changes to the main repository. Let me know what to do.

Sure, my bad mate.
Sadly, as for now I cannot do it, I should solve some issues first :(

If you want we can continue on the fork for the moment just to not get stuck.

My apologies for that unfruitful request, Tanay.

File: ai_engine_sdk/client.py (hunk 2 above)
Comment (XaviPeiro) [reply ai_engine_sdk/client.py]: @tanaygodse Not only in here. Please, notice that there are many changes in the code that are not related to the functionality/fix introduced. Let's keep as clean as possible and avoid those unnecessary changes.

**Review by XaviPeiro — CHANGES_REQUESTED** [review]

File: ai_engine_sdk/client.py (hunk 2 above)
Comment (XaviPeiro) [reply ai_engine_sdk/client.py]: > @XaviPeiro we have back set up and configured on this repo? If not lets add that.

Nope, there is not. I am opening an issue.

File: ai_engine_sdk/api_models/agents_json_messages.py [hunk 3]
```diff
@@ -70,14 +68,16 @@ def is_agent_json_confirmation_message(message_type: str) -> bool:
 
 def is_task_selection_message(message_type: str) -> bool:
     union_of_type = TaskSelectionTypes
-    allowed_values = [literal for lit in get_args(union_of_type) for literal in get_args(lit)]
+    allowed_values = [literal for lit in get_args(
```
Comment (XaviPeiro) [inline ai_engine_sdk/api_models/agents_json_messages.py:71]: here

File: ai_engine_sdk/api_models/agents_json_messages.py [hunk 4]
```diff
@@ -70,14 +68,16 @@ def is_agent_json_confirmation_message(message_type: str) -> bool:
...
+        union_of_type) for literal in get_args(lit)]
     return message_type.upper() in allowed_values
 
 
 def is_data_request_message(message_type: str) -> bool:
     union_of_type = DataRequestTypes
     if get_origin(union_of_type) is Union:
-        allowed_values = [literal for lit in get_args(union_of_type) for literal in get_args(lit)]
```
Comment (XaviPeiro) [inline ai_engine_sdk/api_models/agents_json_messages.py:80]: here

File: ai_engine_sdk/client.py [hunk 5]
```diff
@@ -273,14 +274,15 @@ async def get_messages(self) -> List[ApiBaseMessage]:
                 agent_json: dict = message['agent_json']
                 agent_json_type: str = agent_json['type'].upper()
                 if is_task_selection_message(message_type=agent_json_type):
-                    indexed_task_options: dict = get_indexed_task_options_from_raw_api_response(raw_api_response=message)
+                    indexed_task_options: dict = get_indexed_task_options_from_raw_api_response(
```
Comment (XaviPeiro) [inline ai_engine_sdk/client.py:277]: here and so...

**Review by XaviPeiro — CHANGES_REQUESTED** [review]

File: ai_engine_sdk/client.py (hunk 5 above)
Comment (tanaygodse) [reply ai_engine_sdk/client.py]: Sorry for this formatting issue, I am looking into updating the code formatting in my editor. I have removed all of these formatting issues in my latest push.

**Review by XaviPeiro — APPROVED** [review]

**Pull request merged**